server/services/messaging/message-base.js

```diff
@@ -1,10 +1,11 @@
 const { ServiceBusClient } = require('@azure/service-bus')
 
 class MessageBase {
-  constructor (name, config) {
+  constructor (name, config, credentials, action) {
     this.name = name
     const connectionString = `Endpoint=sb://${config.host}/;SharedAccessKeyName=${config.username};SharedAccessKey=${config.password}`
     this.sbClient = ServiceBusClient.createFromConnectionString(connectionString)
```

Comment:
Missing the code to create the `sbClient` from the token provider if credentials object is not `undefined`

Reply:
Thanks, great spot. Changes pushed.